src/global-settings.php

```diff
@@ -638,11 +662,19 @@ public function form_tag_selector( $tag ) {
 		}
 
 		public function form_paragraph_selector() {
-			return array_merge(
-				$this->form_tag_selector( 'p' ), // Core text.
-				$this->form_tag_selector( 'li' ), // Core lists.
-				$this->form_tag_selector( 'td' ) // Core table cells.
+			$is_apply_body_to_html = get_option( 'stackable_is_apply_body_to_html' ) ?? false;
```

Comment:
I believe `get_option()` is never `null` since it returns `false` if the value is not yet set. So we can do just:
```suggestion
			$is_apply_body_to_html = get_option( 'stackable_is_apply_body_to_html' );
```

Can you also move this code to the `if` statement where it is used to make it closer and easier to read.